src/main/kotlin/no/skatteetaten/aurora/boober/model/openshift/ImageStreamImport.kt

```diff
@@ -0,0 +1,90 @@
+package no.skatteetaten.aurora.boober.model.openshift
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+
+
```

Comment:
Should we note here that this is a partial representation of the ImageStreamImport kind? 
And reference https://github.com/fabric8io/kubernetes-client/issues/1025

Reply:
Yes, I think we should.
Should I just add a comment, or should we name the method is it is understandable?